kafkautils/stream.go

```diff
@@ -38,8 +38,10 @@ func NewSchema(schema string) (*StreamName, error) {
 		Event:     contractEvent[1],
 	}
 
-	aggregation := strings.SplitAfterN(contractEvent[1], TopicAggregateSeparator, 2)
+	aggregation := strings.Split(contractEvent[1], TopicAggregateSeparator)
```

Comment:
They don't do the same thing. cuz `SplitAfterN` keeps the separator while `Split` does not.

Reply:
I think there was a bug in NewSchema function and @cnkarz changed from SplitAfterN to Split to fix it